torch/csrc/jit/runtime/autodiff.cpp

```diff
@@ -49,7 +49,7 @@ bool needTrimGrad(Node* n) {
 bool isDifferentiable(const Node* n) {
   // TODO: scalar-tensor ops should be canonicalized
   static OperatorSet differentiable_ops = {
-      "aten::_slow_conv2d_forward(Tensor self, Tensor weight, int[] kernel_size, Tensor? bias, int[] stride, int[] padding) -> (Tensor, Tensor)",
+      "aten::_slow_conv2d_forward(Tensor self, Tensor weight, int[] kernel_size, Tensor? bias, int[] stride, int[] padding) -> Tensor",
```

Comment:
I think we decided a while back that the TorchScript tracer does not trace into `aten::_convolution`. If that's the case, I don't know why there's a case for `_slow_conv2d_forward` here: I would hope this is just dead code?

Reply:
Seems like we merged https://github.com/pytorch/pytorch/pull/64280 without problem so changes to this file seem like they will be OK

Reply:
Yeah I agree with you - it should be dead code. Notably it's just for `_slow_conv2d_forward`; similar things do not exist for the other slow algos.